eahub/profiles/api/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import mixins
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from eahub.profiles.api.serializers import ProfileSerializer, TagSerializer
+from eahub.profiles.models import Profile, ProfileTag, ProfileTagStatus, ProfileTagType
+
+
+class ProfileViewSet(
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    GenericViewSet,
+):
+    queryset = Profile.objects.filter(user__is_active=True)
+    serializer_class = ProfileSerializer
+
+
+@api_view(["POST"])
+@permission_classes([IsAuthenticated])
+def create_tag_view(request: Response):
+    # todo handle duplicates
```

Comment:
Is this about the "merge" actions we talked? So do you want duplicates to be handled code or rather by admins?

Reply:
here I would need to add better error handling for the sake of UX, for now the user would just be unable to add the tag and wouldn't see any error message, which would be frustrating